sklearn/utils/validation.py

```diff
@@ -1280,7 +1301,7 @@ def _check_sample_weight(sample_weight, X, dtype=None):
     X : {ndarray, list, sparse matrix}
         Input data.
 
-    dtype: dtype, default=None
+    dtype: dtype
```

Comment:
Unrelated to this PR. And we want to keep the default, right ?

Reply:
Yea, I have no idea how this diff got into this PR.

Updated PR reverting this change.